src/Mvc/Mvc.Abstractions/src/ModelBinding/ModelMetadata.cs

```diff
@@ -654,7 +645,7 @@ private void InitializeTypeInformation()
         Debug.Assert(ModelType != null);
 
         IsConvertibleType = TypeDescriptor.GetConverter(ModelType).CanConvertFrom(typeof(string));
-        IsParseableType = ModelMetadata.FindTryParseMethod(ModelType) is not null;
+        IsParseableType = FindTryParseMethod(ModelType) is not null;
```

Comment:
You could consider moving line 651 above this and use `UnderlyingOrModelType` instead.